tests/integration/utils.py

```diff
@@ -463,3 +465,166 @@ def _validate_expected_dep_file_contents(dep_contents_file: Path, output_dir: Pa
         dep_file = output_dir / "deps" / path
         assert dep_file.exists()
         assert dep_file.read_text() == expected_content
+
+
+ALL = object()
+# Individual files could be added to the set as well.
+paths_to_code = set((Path("cachi2"), Path("tests/integration")))
+supported_pms: set[str] = set(
```

Comment:
These are global scoped RO variables -> we should declare them as constants.

Reply:
nitpick: Also, these should go to the top of the file to not break the visual reading continuation - one expects constants at the beginning of the module, then classes, then functions, etc.

Reply:
Good point, this  is a frozenset indeed.

Reply:
nitpick: Can we also use capitals here? This way it still reads more like a global variable or even worse (in context of pytest), as a fixture defined somewhere.

Reply:
Done.